roboflow/__init__.py

```diff
@@ -59,20 +59,13 @@ def check_key(api_key, model, notebook, num_retries=0):
         return "onboarding"
 
 
-def auth(api_key):
```

Comment:
It is backward incompatible. I'd prefer to keep this function.

Reply:
Okay, but I am curious about the use case, at this moment the calling of this function raises an exception
`TypeError: auth() missing 1 required positional argument: 'api_key'`

Reply:
Ah ok , I didn't realized it is totally broken. 👍🏻 Yes, we can remove it.

Reply:
Can you remove it again? 😅 I reviewed all the PR already (all good), I'll wait for this last change.

Reply:
Sure! Done!